README.md

```diff
@@ -53,8 +53,20 @@ The tools available in FRE-NCtools are:
 * runoff_regrid -- Regrid land runoff data to the nearest ocean point
 * transfer_to_mosaic_grid -- Convert older style grids to newer mosaic grid
 
+### Other tools
+The [ Ocean Model Grid Generator](https://github.com/NOAA-GFDL/ocean_model_grid_generator)
```

Comment:
Please remove the space after the `[`.  I usually wouldn't be this specific, but as there are other comments, might as well address this one as well.

Reply:
All improvements are appreciated!